projects/core/src/global-message/http-interceptors/handlers/unknown-error/unknown-error.handler.ts

```diff
@@ -1,16 +1,25 @@
 import { Injectable, isDevMode } from '@angular/core';
 import { HttpResponseStatus } from '../../../models/response-status.model';
 import { HttpErrorHandler } from '../http-error.handler';
+import { Priority } from '../../../../util/handler';
 
 @Injectable({
   providedIn: 'root',
 })
 export class UnknownErrorHandler extends HttpErrorHandler {
   responseStatus = HttpResponseStatus.UNKNOWN;
 
+  hasMatch(_errorResponse): boolean {
+    return true;
+  }
+
   handleError() {
     if (isDevMode()) {
       console.warn(`Unknown http response error: ${this.responseStatus}`);
     }
   }
+
+  getPriority() {
+    return Priority.FALLBACK;
```

Comment:
can we add some docs to explain why this handler has a lower priority?

Reply:
👍